client/src/components/main_section/main_section.coffee

```diff
@@ -30,6 +31,7 @@ MainSection = React.createClass
               Home {}
           when 'help' then Help {}
           when 'login' then LoginForm { loginActions }
+          when 'registration' then RegisterForm { registerActions }
```

Comment:
I would start extracting this switch case into a function that returns a thunk of the correct component